clients/osv/src/main/kotlin/Model.kt

```diff
@@ -75,6 +75,25 @@ data class Credit(
     val contact: List<String> = emptyList()
 )
 
+/**
+ * Defined package ecosystem values, see https://ossf.github.io/osv-schema/.
+ */
+object Ecosystem {
```

Comment:
Could you please explain in the commit message why this is not an enum?

I *guess* this because you don't want to break deserialization on unknown values, and don't want to put effort into a custom deserializer...